### PR TITLE
TSS2 RAV4 firmware versions

### PR DESCRIPTION
**Pull request by jmedzen**
update fingerprint for Car.RAV4_TSS2.

### CODE REVIEW

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -1057,6 +1057,7 @@ class CAR:
   },
   CAR.RAV4_TSS2: {
     (Ecu.engine, 0x700, None): [
+      b'\0018966342V3000\000\000\000\000',
```
Comment (vanillagorillaa) [inline selfdrive/car/toyota/values.py:1060]: This should be moved down and arranged in order with the other firmware already there. Also the format should be \x00\x00

File: selfdrive/car/toyota/values.py [hunk 2]
```diff
@@ -1084,6 +1085,7 @@ class CAR:
       b'\x02896634A47000\x00\x00\x00\x00897CF4201001\x00\x00\x00\x00',
     ],
     (Ecu.esp, 0x7b0, None): [
+      b'\001F152642561\000\000\000\000\000\000',
```
Comment (vanillagorillaa) [inline selfdrive/car/toyota/values.py:1088]: This one isn't needed, it's already in the lis.

File: selfdrive/car/toyota/values.py [hunk 3]
```diff
@@ -1096,6 +1098,7 @@ class CAR:
       b'\x01F152642750\x00\x00\x00\x00\x00\x00',
     ],
     (Ecu.eps, 0x7a1, None): [
+      b'8965B42170\000\000\000\000\000\000',
```
Comment (vanillagorillaa) [inline selfdrive/car/toyota/values.py:1101]: Not needed, it's already in the list

File: selfdrive/car/toyota/values.py [hunk 4]
```diff
@@ -1105,12 +1108,14 @@ class CAR:
       b'\x028965B0R01400\x00\x00\x00\x008965B0R02400\x00\x00\x00\x00',
     ],
     (Ecu.fwdRadar, 0x750, 0xf): [
+      b'\0018821F3301200\000\000\000\000',
```
Comment (vanillagorillaa) [inline selfdrive/car/toyota/values.py:1111]: Not needed, it's already in the list

File: selfdrive/car/toyota/values.py [hunk 5]
```diff
@@ -1105,12 +1108,14 @@ class CAR:
...
       b'\x018821F3301100\x00\x00\x00\x00',
       b'\x018821F3301200\x00\x00\x00\x00',
       b'\x018821F3301300\x00\x00\x00\x00',
       b'\x018821F3301400\x00\x00\x00\x00',
     ],
     (Ecu.fwdCamera, 0x750, 0x6d): [
+      b'\0028646F4203500\000\000\000\0008646G2601200\000\000\000\000',
```
Comment (vanillagorillaa) [inline selfdrive/car/toyota/values.py:1118]: Not needed, it's already in the list

**Comment by vanillagorillaa** [comment]
Please also include the dongleID from the device this change came from (Settings -> Device -> DongleID)

File: selfdrive/car/toyota/values.py (hunk 1 above)
Comment (chicman) [reply selfdrive/car/toyota/values.py:1060]: Those \000 is from my log file, looked into values.py I was confused by \000 \x00 also. 
I tried to change them into /x00 and OP still works!

I'll resubmit another pull with proper info!
Best Wishes to all of u!

**Comment by chicman** [comment]
> Please also include the dongleID from the device this change came from (Settings -> Device -> DongleID)

Which file or place should I put the dongleID?

**Comment by vanillagorillaa** [comment]
Just post the dongleID here in the comments

**Comment by chicman** [comment]
> Just post the dongleID here in the comments

dongleID: 7c8e5c7cebab8bc9

if this is good for u to accept the firmware ID? Or I will create another pull!

**Comment by vanillagorillaa** [comment]
Update this PR with the above changes.

**Comment by chicman** [comment]
dongleID: 7c8e5c7cebab8bc9

**Comment by VirtuallyChris** [comment]
Merged! Thanks for the pull request!

**Pull request merged**